src/datetime.js

```diff
@@ -1073,6 +1074,17 @@ export default class DateTime {
     return this.isValid ? daysInYear(this.year) : NaN;
   }
 
+  /**
+   * Returns the number of weeks in this DateTime's year
+   * @see https://en.wikipedia.org/wiki/ISO_week_date
+   * @example DateTime.local(2004).weeksInWeekYear //=> 53
+   * @example DateTime.local(2013).weeksInWeekYear //=> 52
+   * @type {number}
+   */
+  get weeksInWeekYear() {
+    return this.isValid ? weeksInWeekYear(this.year) : NaN;
```

Comment:
`this.year` isn't right, and will break for dates with different ISO and Gregorian years. You need `this.weekYear`.